src/main/java/org/spongepowered/api/text/channel/MessageReceiver.java

```diff
@@ -135,4 +127,18 @@ default Locale getLocale() {
         return Locales.DEFAULT;
     }
 
+    /**
+     * Represents a {@link MessageReceiver} that can have their
+     * {@link MessageChannel} changed.
```

Comment:
So could you clarify here the uniqueness of what this means? That a Mutable variation can still only naturally send to one single channel (which can be comprised of multiple channels), and some will only send to a specific channel?

Is it still possible that there's events for the message sent to "forward" messages to other channels?